capture/capture.yaml

```diff
@@ -83,16 +83,23 @@ components:
             The byte offset to filter on
           type: integer
           default: 0
+        bit_length:
+          description: >-
+            The bit length to filter on
```

Comment:
The bit length of field to filter on

Reply:
change the description for byte offset as well. "The byte offset of field to filter on"

Reply:
updated